articles/cosmos-db/mongodb-feature-support.md

```diff
@@ -23,14 +23,19 @@ Azure Cosmos DB is Microsoft's globally distributed multi-model database service
 
 By using the Azure Cosmos DB MongoDB API, you can enjoy the benefits of the MongoDB APIs you're used to, with all of Azure Cosmos DB's enterprise capabilities: [global distribution](distribute-data-globally.md), [automatic sharding](partition-data.md), availability and latency guarantees, automatic indexing of every field, encryption at rest, backups, and much more.
 
+## MongoDB Protocol Support
+
+The Cosmos DB MongoDB API is compatible with MongoDB Server version **3.2** by default. The supported operators and any limitations or exceptions are listed below. Features or query operators added in MongoDB version **3.4** are currently available as a preview feature. Any client driver that understands these protocols should be able to connect to Cosmos DB using the MongoDB API.
```

Comment:
The style guide recommends to call out the full name "Azure Cosmos DB" at each occurrence because there is an internal product with name Cosmos and we don't want users to get confused with this name.